apiserver/uniter/uniter_test.go

```diff
@@ -2755,11 +2763,391 @@ func (s *uniterNetworkConfigSuite) TestNetworkConfigForImplicitlyBoundEndpoint(c
 
 	expectedConfig := []params.NetworkConfig{{Address: privateAddress.Value}}
 
-	result, err := s.base.uniter.NetworkConfig(args)
+	result, err := s.base.uniterv3.NetworkConfig(args)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(result, jc.DeepEquals, params.UnitNetworkConfigResults{
 		Results: []params.UnitNetworkConfigResult{
 			{Config: expectedConfig},
 		},
 	})
 }
+
+type uniterNetworkInfoSuite struct {
+	base uniterSuite
+}
+
+var _ = gc.Suite(&uniterNetworkInfoSuite{})
+
+func (s *uniterNetworkInfoSuite) SetUpSuite(c *gc.C) {
+	s.base.SetUpSuite(c)
+}
+
+func (s *uniterNetworkInfoSuite) TearDownSuite(c *gc.C) {
+	s.base.TearDownSuite(c)
+}
+
+func (s *uniterNetworkInfoSuite) SetUpTest(c *gc.C) {
+	s.base.JujuConnSuite.SetUpTest(c)
```

Comment:
It's completely based on NetworkConfig suite, I'll add a ticket to clean up the whole uniter_test.